README.md

```diff
@@ -1,2 +1,111 @@
+        docker_push: 'true'
+        sonar_scan_enabled: 'false'
+
+```
+
+## parameters
+
+### Input Parameters
+
+
+
+| Parameter                 | Default               | Description                                          | Required |  Sample input        |
+|---------------------------|-----------------------|------------------------------------------------------|----------|----------------------|
+| `src_file_dir_path`       | `.`                   | Directory of the solution file                       | true     |'./' , './src/test'   |
+| `dockerfile_dir_path`     | `.`                   | Directory of the dockerfile                          | true     |'./' , './src/test'   |
+| `github_token`            |                       | Github token.Pass same secret as sample snippet      | true     |                      |
+| `github_owner_token`      |                       | Github owner token.Pass same secret as sample snippet| true     |                      |
+| `sonar_token`             |                       | Sonar token.Pass same secret as sample snippet.      | true     |                      |
+| `aws_access_key_id`       |                       | aws access key id.Pass same secret as sample snippet.| true     |                      |
+| `aws_secret_access_key`   |                       | aws secret access key.Same secret as sample snippet. | true     |                      |
+| `aws_region`              |                       | aws region.Pass same secret as sample snippet.       | false    |                      |
+| `docker_repo_name`        |                       | docker repository name.                              | true     |                      |
+| `docker_image_name`       |                       | docker image name.                                   | true     |                      |
+| `sonar_project_key`       |   `variant-inc`       | sonar project key.                                   | false    |                      |
+| `sonar_org`               |   `variant`           | sonar organization.                                  | false    |                      |
+| `sonar_scan_enabled`      |    `false`            | set to true if sonar scan enabled or set to false.   | false    |                      |
+| `nuget_push`              |    `false`            | set to true if nuget push is enabled or set to false.| false    |                      |
+| `docker_push`             |    `false`            | set to true if push to ECR is enabled set to false.  | false    |                      |
+
+
+### Output Parameters
+
+Output parameters section.
+
+```text
+image:
+Image tag will have docker image name generated.
+
+```
+
+Output parameters can be invoked as mentioned in below snippet.
+
+
+```yaml
+
+    - name: Get the image name from lazy-action
+      run: echo "Image Name:${{ steps.lazy-action.outputs.image }}"
+
+```
+steps.lazy-action.outputs.image :
+
+* lazy-action : id from your lazy action invocation.
```

Comment:
What is this ID for?

Reply:
Just referencing id from the dotnet action steps to get the output prameter.